wemake_python_styleguide/visitors/ast/imports.py

```diff
@@ -79,14 +81,17 @@ def validate(self, node: ast.Import) -> None:
 
     def _check_dotted_raw_import(self, node: ast.Import) -> None:
         for alias in node.names:
-            if '.' in alias.name:
+            if MODULE_MEMBERS_SEPARATOR in alias.name:
                 self._error_callback(
                     DottedRawImportViolation(node, text=alias.name),
                 )
 
     def _check_protected_import(self, node: ast.Import) -> None:
         names: Iterable[str] = chain.from_iterable(
```

Comment:
We can write this line more compact as:

```python
names: Iterable[str] = chain.from_iterable([
    alias.name.split(MODULE_MEMBERS_SEPARATOR)
    for alias in node.names
])
```

This would be an extra rule for our `libcst` based visitor.

Reply:
will this rule be related only to calls with single argument?

Reply:
I am not sure yet. It is not yet formalised 😞